src/templates/Component.ts

```diff
@@ -0,0 +1,22 @@
+import { DNode } from '@dojo/widget-core/interfaces';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { ThemeableMixin, ThemeableProperties, theme } from '@dojo/widget-core/mixins/Themeable';
+import { v } from '@dojo/widget-core/d';
+
+import * as css from './<%= componentStylePath %>';
+
+/**
+ * @type <%= name %>Properties
+ *
+ * Properties that can be set on <%= name %> components
+ */
+export interface <%= name %>Properties extends ThemeableProperties { };
```

Comment:
Don't need to extend `ThemeableProperties`

Reply:
Do I not need to explicitly extend `ThemeableProperties` for a themeable widget? This generated component is themeable and specific to UI components (for now). We figured future versions would include a flag to disable any theme or CSS related generation at all for non-UI components.

Reply:
Correct, you don't need to explicitly extend the `ThemeableProperties` - just an FYI in master `ThemeableProperties` has been renamed `ThemedProperties` and `ThemeableMixin` renamed to `ThemedMixin`

Reply:
Just using the `ThemedMixin` automagically ensures that the `ThemedProperties` are available on the widgets properties interface.

Reply:
Updated, thanks.